invenio_rdm_records/cli.py

```diff
@@ -69,14 +72,33 @@ def fake_date(end_date=None):
     return f_date
 
 
-def create_fake_record():
+def create_fake_record(access_status):
     """Create records for demo purposes."""
     fake = Faker()
     data_to_use = {
         "access": {
-            "record": "public",
-            "files": "public",
-        },
+            RecordAccessStatusEnum.open: {
+                "record": "public",
+                "files": "public",
+            },
+            RecordAccessStatusEnum.embargoed: {
+                "record": "public",
+                "files": "restricted",
+                "embargo": {
```

Comment:
Conflicts with https://github.com/inveniosoftware/invenio-app-rdm/issues/717. It can be removed, added to make it easier to try out.